test/test_arraycontext.py

```diff
@@ -291,8 +295,9 @@ def get_imag(ary):
             (operator.neg, 1, False),
             (operator.abs, 1, False),
 
-            (get_real, 1, False),
-            (get_imag, 1, False),
+            # Not supported in pytato:
+            # (get_real, 1, False),
+            # (get_imag, 1, False),
```

Comment:
Wouldn't this decrease test coverage for the pyopencl actx?

Reply:
Added real/imag support to pytato, restored test.